pipelines/push-disk-images-to-cdn/README.md

```diff
@@ -20,6 +20,9 @@ Tekton Pipeline to push disk images to a cdn using pulp
 | taskGitUrl                      | The url to the git repo where the release-service-catalog tasks to be used are stored                  | Yes | https://github.com/konflux-ci/release-service-catalog.git |
 | taskGitRevision                 | The revision in the taskGitUrl repo to be used                                                         | No       | -               |
 
+## Changes in 1.2.0
+* Add new param `schema` to `check-dataz-keys` task to allow for a JSN schema to be passed in.
```

Comment:
typo dataz

Reply:
typo JSN

Reply:
(These are in the other READMEs too)

Reply:
Removed the pipeline see: https://github.com/konflux-ci/release-service-catalog/pull/603#discussion_r1801148486